lib/VerifiableCredentialStore.js

```diff
@@ -84,14 +97,15 @@ export default class VerifiableCredentialStore {
   /**
    * Stores a verifiable credential in remote private storage
    *
-   * @param {Object} credential
+   * @param {object} credential
+   * @param {object} meta
    */
-  async insert({credential}) {
+  async insert({credential, meta}) {
```

Comment:
Is the new meta parameter required parameter?  I'm concerned about the lack of `undefined` check here and it is assumed that an object is present and properties are added.

We'd need to be sure to release this as a breaking release if so.

Reply:
Yes it will need to be required going forward.